src/ansys/fluent/core/utils/file_transfer_service.py

```diff
@@ -150,37 +151,40 @@ def __init__(
         ----------
         image_name: str
             Name of the image.
+        image_tag : str, optional
+            Tag of the image.
         port: int, optional
-            Port for file transfer service to use.
+            Port for the file transfer service to use.
         container_mount_path: Union[str, Path], optional
             Path inside the container where host mount path will be mounted to.
```

Comment:
```suggestion
            Path inside the container to mount the host mount path to.
```

Reply:
@PipKat I am not sure how, it is literally "mounting" the "host mount path" inside the container. Maybe:
```suggestion
            Path inside the container for the host mount path.
```
Leaving the fact that it will be mounted implied (should be enough I think).

Reply:
Done. Thank you.